### ISSUE TITLE
Single timestamp for datasink

### ISSUE THREAD

**Issue by daradermody** [regel/loudml#76]
Is it possible that the prediction results outputted from each model to a single datasink could all use the same time field? Currently, it uses the time field name of each individual source, which means that I can not easily do date histograms with term aggregations on the anomalies from different models.

**Comment by regel** [comment]
Hi @daradermody , can you please dump 1 example to make sure we have the right understanding?

**Comment by daradermody** [comment]
Here is sample output from two models that are using the same datasink:
```json
{
  "_index": "ml-output-2008.09.29",
  "_type": "doc",
  "_id": "9AbaLGcBJxd3hxcoBsv2",
  "_score": 1,
  "_source": {
    "0": 0,
    "lower_0": 0,
    "upper_0": 0,
    "model": "company-count",
    "score": 0,
    "founded_date": 1222646400000,
    "is_anomaly": true
  }
},
{
  "_index": "ml-output-2018.11.07",
  "_type": "doc",
  "_id": "bAb9K2cBJxd3hxcoG4ub",
  "_score": 1,
  "_source": {
    "value": 0.23169922828674316,
    "lower_value": 0.09199491814153604,
    "upper_value": 0.37140353843195006,
    "model": "sine-wave",
    "score": 5.194049802662493,
    "@timestamp": 1541571627000,
    "is_anomaly": false
  }
},
```

The datasource for one model uses `@timestamp` and the other uses `founded_date` as time fields. As a result, the timestamps for each are different. In Kibana and Siren's platform, you choose one date field as the primary time field, which is used in dashboards and visualisations for that index pattern:

![image](https://user-images.githubusercontent.com/7104356/48723369-5a1faf00-ec1e-11e8-915a-6e5f4c7bc316.png)

Not being able to choose one timefield that encompasses all of the Loud ML datasink results makes it difficult to create dashboards, or use the data from different models dynamically.

**Comment by regel** [comment]
Okay thanks, got it. So, do we keep the original date field name (..any use-cases for that?) or do we use a standard date field name and apply it to all sinks in order to keep consistency in the data model (and mapping) written to the TSDB?

It's easier when the TSDB does not have the luxury to change the date field names :)

**Comment by daradermody** [comment]
The latter would solve the primary time field issue. I can't think of any use case for keeping the original time field in the datasink.

**Comment by regel** [comment]
@daradermody this can be a good first issue to submit a PR. See elastic.py source code.

**Comment by regel** [comment]
Waiting for this [PR](https://github.com/regel/loudml/pulls) to be merged. The timestamp field is moved to the bucket definition.

**Comment by regel** [comment]
Solved on master (future 1.6.0 release). The timestamp field for Elasticsearch is now defined in the bucket schema.
